src/viperleed/utilities/poscar/modify_vacuum.py

```diff
@@ -127,10 +143,15 @@ def parse_cli_args(self, args):
 
     def process_slab(self, slab, args):
         """Return a new slab with modified vacuum."""
+        vacuum_gap_info = VacuumGapInfo(
+            size=args.vacuum,
+            absolute=args.absolute,
+            accept_small_gap=args.force,
```

Comment:
We don't seem to be testing the `accept_small_gap == True` case. Also, `parse_cli_args` is only tested for the "failing" case.